flytekit/configuration/aws.py

```diff
@@ -21,3 +21,5 @@
 RETRIES = _config_common.FlyteIntegerConfigurationEntry("aws", "retries", default=3)
 
 BACKOFF_SECONDS = _config_common.FlyteIntegerConfigurationEntry("aws", "backoff_seconds", default=5)
+
+FAST_REGISTRATION_DIR = _config_common.FlyteStringConfigurationEntry("aws", "fast_registration_dir")
```

Comment:
why is this under aws, i think it should be under common config?

Reply:
moved